Examples/MultiModal/Mixed_Image__And_Tabular_Reading/CompositeReader.ipynb

```diff
@@ -0,0 +1,429 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Composite Data Readers Tutorial**\n",
```

Comment:
This notebook is looking pretty good. We would like to merge this with the https://github.com/Microsoft/CNTK/blob/master/Manual/Manual_How_to_feed_data.ipynb

Could you please make the changes?